docs/users.rst

```diff
@@ -1248,8 +1248,10 @@ available in the OSBS buildroot as `bsi` executable in `$PATH`.
 
 
 Current limitations:
-* only Source RPMs are added into source container image
+
+* only Source RPMs and sources fetched through :ref:`cachito-integration` are added into source container image
```

Comment:
We should substitute the "*" for "-" in both lines here: This is how lists in rst work.

Reply:
NVM, maybe it was broken due to the missing line break!